src/wweb/onMessage/commands.ts

```diff
@@ -13,8 +13,19 @@ import quoteMessage from './meme/quoteMessage';
 import tigerMessage from './meme/tigerMessage';
 import tradeOfferMessage from './meme/tradeOfferMessage';
 import gptConfigContextMessage from './gpt/gptConfigContextMessage';
+import { Message } from 'whatsapp-web.js';
+import githubMessage from './githubMessage';
 
-const COMMANDS = {
+type Commands = {
+  [key: string]: {
+    name: string;
+    description?: string;
+    example?: string;
+    action: (msg: Message) => Promise<unknown>;
+  };
+}
```

Comment:
```suggestion
type Commands = Record<string, {
    name: string;
    description?: string;
    example?: string;
    action: (msg: Message) => Promise<unknown>;
  }>
```

Reply:
we can might define the commands as well to be fully type-safe

Reply:
yes, we can do it in another pull request, it is a good improvement